api/services/inventory/sock_store.go

```diff
@@ -204,3 +204,62 @@ func (s *SockStore) GetSockByID(sockID int) (*types.Sock, error) {
 
 	return &sock, nil
 }
+
+// UpdateSock updates an existing sock and its variants in the database
+func (s *SockStore) UpdateSock(sockID int, sock types.Sock, variants []types.SockVariant) error {
+	// Update the sock details
```

Comment:
nit; you can remove all of the comments inside the function body, I think the code explains itself

Reply:
Fixed!